lib/src/utils/core.dart

```diff
@@ -47,26 +47,25 @@ import 'package:ox_coi/src/l10n/l10n.dart';
 
 enum ProtocolType { imap, smtp }
 
```

Comment:
As we should keep the "automatic" value, so letting this empty, we should initialize both methods with the empty case.